samples/README.md

```diff
@@ -1,28 +1,56 @@
-# Sample apps for the AWS IoT Device SDK for C++ v2
-
-## MQTT5 Samples
-#### MQTT5 is the recommended MQTT Client. It has many benefits over MQTT311 outlined in the [MQTT5 User Guide](../documents/MQTT5_Userguide.md)
-* [Mqtt5 Pub-Sub](./mqtt5/mqtt5_pubsub/README.md)
-    + [Direct MQTT with X509-based mutual TLS](./mqtt5/mqtt5_pubsub/README.md#direct-mqtt-with-x509-based-mutual-tls)
-    + [MQTT over Websockets with Sigv4 authentication](./mqtt5/mqtt5_pubsub/README.md#mqtt-over-websockets-with-sigv4-authentication)
-    + [Direct MQTT with Custom Authentication](./mqtt5/mqtt5_pubsub/README.md#direct-mqtt-with-custom-authentication)
-    + [MQTT over Websockets with Cognito](./mqtt5/mqtt5_pubsub/README.md#mqtt-over-websockets-with-cognito)
-    + [HTTP Proxy](./mqtt5/mqtt5_pubsub/README.md#http-proxy)
-## Other Samples
-* [Jobs Sandbox](./jobs/jobs-sandbox/README.md)
-* [Shadow Sandbox](./shadow/shadow-sandbox/README.md)
-* [Basic Fleet Provisioning](./fleet_provisioning/provision-basic/README.md)
-* [CSR Fleet Provisioning](./fleet_provisioning/provision-csr/README.md)
-* [Commands Sandbox](./commands/commands-sandbox/README.md)
-* [Secure Tunnel](./secure_tunneling/secure_tunnel/README.md)
-* [Secure Tunnel Notification](./secure_tunneling/tunnel_notification/README.md)
-* [Cycle Pub-Sub](./pub_sub/cycle_pub_sub/README.md)
-* [Greengrass discovery](./greengrass/basic_discovery/README.md)
-* [Greengrass IPC](./greengrass/ipc/README.md)
-* [Mqtt5 Device Defender](./device_defender/mqtt5_basic_report/README.md)
-* [Mqtt311 Device Defender](./device_defender/basic_report/README.md)
-
-## Build Instruction
+# Sample for the AWS IoT Device SDK v2 for C++
+This directory contains sample applications for [aws-iot-device-sdk-cpp-v2](../README.md).
+
+### Table of Contents
+* [Samples](#samples)
+    * [MQTT5 Client Samples](#mqtt5-client-samples)
+    * [Service Client Samples](#service-client-samples)
+    * [Greengrass Samples](#greengrass-samples)
+* [Instructions](#instructions)
+* [Sample Help](#sample-help)
+* [Enable Logging](#enable-logging)
+
+
+## Samples
+### MQTT5 Client Samples
+##### MQTT5 is the recommended MQTT Client. Additional information and usage instructions can be found in the [MQTT5 User Guide](../documents/MQTT5_Userguide.md). The samples below will create an MQTT5 client, connect using the selected method, subscribe to a topic, publish to the topic, and then disconnect.
+| MQTT5 Client Sample | Description |
+|--------|-------------|
+| [X509-based mutual TLS](./mqtt/mqtt5_x509/README.md) | Demonstrates connecting to AWS IoT Core using X.509 certificates and private keys.
+| [Websockets with Sigv4 authentication](./mqtt/mqtt5_aws_websocket/README.md) | Shows how to authenticate over websockets using AWS Signature Version 4 credentials. |
+| [AWS Custom Authorizer Lambda Function](./mqtt/mqtt5_custom_auth_signed/README.md) | Examples of connecting with a signed and unsigned Lambda-backed custom authorizer.
```

Comment:
Since this has two separate README files for signed/unsigned we should add separate links to both types here.